website/api_versioned_sidebars/version-1.2-sidebars.json

```diff
@@ -148,8 +148,8 @@
         },
         {
           "type": "doc",
-          "label": "Cryptography Acceleration",
-          "id": "zkvm/acceleration"
+          "label": "Precompiles",
+          "id": "zkvm/precompiles"
```

Comment:
We will probably want to create a redirect to avoid 404s for users who end up at the `zkvm/acceleration` page -- i don't recall the details of how those are set up but we can ask cohan/morgan for help if necessary